_docs/tasks/telemetry/fluentd.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Logging with Fluentd
+
+overview: This task shows you how to configure Istio to log to a Fluentd daemon
+
+order: 60
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This task shows how to configure Istio to create custom log entries
+and send them to a Fluentd daemon. At the end of this task, a new log
+stream will be enabled sending logs to an example Fluentd /
+Elasticsearch / Kibana stack.
+
```

Comment:
Do we want any one/two sentence introduction about the role of the three systems involved (fluentd for logs export, Elasticsearch for logs processing, Kibana for UI)?